src/carousel/carousel.ts

```diff
@@ -85,6 +87,12 @@ export class NgbCarousel implements AfterContentChecked,
    */
   @Input() activeId: string;
 
+  /**
+   * A carousel slide event fired when the slide transition is completed.
+   * See NgbSlideCarouselEvent for payload details
+   */
+  @Output() onCarouselSlide = new EventEmitter<NgbSlideCarouselEvent>();
```

Comment:
can we name it just `slide`? We are in the carousel context so no need to add the `Carousel` in the name. Similarly we know that this is an event so no need for `on` (we are not adding it for other directives / events).